Classes/Fusion/Helper/NodeInfoHelper.php

```diff
@@ -217,17 +217,23 @@ protected function getUriInformation(NodeInterface $node, ControllerContext $con
      */
     protected function getBasicNodeInformation(NodeInterface $node): array
     {
+        $parent = $node->getParent();
+        $isAutoCreated = $node->isTethered();
+        $parentConfiguration = $parent ? $parent->getNodeType()->getFullConfiguration() : [];
+        $isHidableAutoCreatedChild = (bool)$parentConfiguration['childNodes'][(string)$node->getNodeName()]['hideable'] ?? false;
+        $policyForbidsHiding = in_array('_hidden', $this->nodePolicyService->getDisallowedProperties($node));
         return [
             'contextPath' => $node->getContextPath(),
             'name' => $node->getName(),
             'identifier' => $node->getIdentifier(),
             'nodeType' => $node->getNodeType()->getName(),
             'label' => $node->getLabel(),
-            'isAutoCreated' => $node->isAutoCreated(),
+            'isAutoCreated' => $isAutoCreated,
+            'disableChangeVisibility' => $policyForbidsHiding || ($isAutoCreated && $isHidableAutoCreatedChild === false),
```

Comment:
we should probably use a non negated variable name like [`isHidable`
](https://english.stackexchange.com/questions/115129/what-other-words-can-be-used-instead-of-hideable#115141) (with inverted value) as a more descriptive name than: `disableChangeVisibility`.